css/resume.css

```diff
@@ -0,0 +1,184 @@
+/*!
+ * Start Bootstrap - Resume v5.0.8 (https://startbootstrap.com/template-overviews/resume)
+ * Copyright 2013-2019 Start Bootstrap
+ * Licensed under MIT (https://github.com/BlackrockDigital/startbootstrap-resume/blob/master/LICENSE)
+ */
+
```

Comment:
smart using bootstrap for such a time sensitive assignment! I haven't had a chance to dig into this yet.